eva.py

```diff
@@ -199,16 +198,75 @@ def crop_edge_times(da, drop):
     return da.isel({'time': slice(start, stop)})
 
 
+def fix_output_metadata(da, mode, freq, extreme_type, outvar=None, season=None, month=None):
+    """Edit xclim default output variable attributes.
+
+    Parameters
+    ----------
+    da : xarray DataArray
+    mode : {'min', 'max'}
+        Block reduction operation (i.e. block maxima or minima)
+    freq : str
+        Time frequency for blocks
+    extreme_type : {'ari', 'aep', 'quantile'}
+        Type of extreme to return
+        ari = average recurrence interval
+        aep = annual exceedance probability
+    outvar : str, optional
+        Name of custom output variable
+    month : list, optional
+        Restrict analysis to these months (list of month numbers)
+    season : {'DJF', 'MAM', 'JJA', 'SON'}, optional
+        Restrict analysis to a particular season
+
+    """
+
+    del da.attrs['cell_methods']
+    del da.attrs['method']
+    del da.attrs['history']
+    da.attrs['long_name'] = da.attrs['original_long_name']
+    del da.attrs['original_long_name']
+
+    da.attrs['block_frequency'] = freq
+    extreme_type_text = extreme_type.upper() if extreme_type in ['ari', 'aep'] else extreme_type
+    da.attrs['description'] = da.attrs['description'].replace('Quantile', extreme_type_text)
+    if mode == 'min':
+        da.attrs['direction'] = 'probability of non-exceedance'
+        da.attrs['long_name'] = 'Block Minima of ' + da.attrs['long_name']
+    elif mode == 'max':
+        da.attrs['direction'] = 'probability of exceedance'
+        da.attrs['long_name'] = 'Block Maxima of ' + da.attrs['long_name']
```

Comment:
@tonyrafter - I like your suggestion of linking `long_name` to the `mode` rather than trying to keep a list of all the different possible output variable names that people might like to use. Given that we're implementing the "block maxima" or "block minima" method of extreme value analysis, I figure saying "Block Maxima" or "Block Minima" in the long name is the most technically correct way to do it.